api/aecworks/settings.py

```diff
@@ -69,6 +69,7 @@
     "django_extensions",
     "debug_toolbar",
     "django_celery_results",
+    "django_cleanup",
```

Comment:
ext. auto deletes images from bucket when the corresponding model is deleted
https://github.com/un1t/django-cleanup